### ISSUE TITLE
Can't delete cluster (GKE)

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#10783]
**Rancher versions:** 2.0 master 1/8

**Steps to Reproduce:**
1.  Create a GKE cloud cluster
2.  After it is fully setup, try to remove the cluster

**Results:**  Gets stuck in removing - Error "Waiting on controller.cattle.io/cluster-provisioner-controller"

**Comment by tfiduccia** [comment]
Closing this in favor of #10805 - it includes both cloud and rke clusters and new error message
